qiskit/circuit/library/standard_gates/xx_minus_yy.py

```diff
@@ -154,7 +154,15 @@ def _define(self):
         self.definition = circuit
 
     def inverse(self, annotated: bool = False):
-        """Inverse gate."""
+        """Inverse gate.
+
+        Args:
+            annotated: indicates whether the inverse gate can be implemented
+                as an annotated gate.
```

Comment:
```suggestion
            annotated: Typically when set to ``True`` this is used to return an :class:`.AnnotatedOperation` 
                with an inverse modifier set instead of a concrete :class:`.Gate`. However, for this class this
                argument is ignored as this inverse of this gate is always a :class:`.XXMinusYYGate` with inverse
                parameter values.
```